extensions/amp-story/1.0/test/validator-amp-story-cta-layer-error.out

```diff
@@ -59,13 +59,18 @@ amp-story/1.0/test/validator-amp-story-cta-layer-error.html:54:6 Tag 'amp-story-
 |        </amp-story-cta-layer>
 |        <amp-story-grid-layer template="vertical">
 |          <h1>fill</h1>
+|          <amp-story-cta-layer animate-in="fade-in">
```

Comment:
I would like to confirm that we expect this to be an error (yeah, it's the error test file...but)?

Reply:
Yes, we just want to make sure that they users can't animate the `amp-story-cta-layer` tag, only descendants of it.

Reply:
This looks like it should fail, but with a different error, right? `<amp-story-cta-layer>` shouldn't be inside of `<amp-story-grid-layer>` here

Reply:
You're right. Just fixed it.